SYSTEMS.md

```diff
@@ -41,14 +83,61 @@ The "rows" property must contain an array of an array of column data.
 Inside the row array, the plugin expects items in the following format:
 ```json
 {
-  "name": <COLUMN_NAME>,
-  "type": <DATA_TYPE>,
-  "coltype": <COLUMN_TYPE>,
-  "value": <DATA_VALUE>
+  "name": "<COLUMN_NAME>",
+  "type": "<DATA_TYPE>",
+  "coltype": "<COLUMN_TYPE>",
+  "value": "<DATA_VALUE>"
 }
 ```
 
-__Each item corresponds to a single column on the sheet. If you have multiple rows, then each row must have the same number of columns.__ See the below Examples section for more examples.
+Therefore, a one row array should look like:
+```json
+ "rows": [
+  [
+  {
+
+  },
+  {
+
+  }
+  ]
+] 
+```
+
+Two rows should look like:
+```json
+ "rows": [
+  [
+    {
+
+    },
+    {
+
+    }
+  ],
+  [
+    {
+
+    },
+    {
+
+    }
+  ]
+] 
+```
+
+You can have more than two rows.
+
+__Each item corresponds to a single column on the sheet. If you have multiple rows, then each row must have the same number of columns.__ See the below Examples section for more examples (and note, in the above structure example, the two row example has 2 columns in each). In order to achieve this, you may have to make empty columns:
+```json
+  {
+    "name": ".",
+    "type": "direct",
+    "coltype": "skip",
+    "value:" ""
+  }, // no comma if the last column
+```
+Note that you can't name all your empty columns the same. Try ".", "..", "...", etc. Since "coltype": is "skip" it won't show the periods, and you can easily count if you've made enough empty columns to match each row. It is worth mentioning here that no two columns can have the same "name": value.
```

Comment:
```suggestion
Note that you can't even empty columns need unique names. Feel free to be as descriptive as necessary, since the column **name** will not be displayed if the `coltype` is `skip`.  **No two columns can have the same `name` value.**
```

Reply:
Should be "Note" that even strike out "you can't"

Reply:
> Should be "Note" that even strike out "you can't"

Yeah it should be "Note: Even empty columns need unique names."  
Sorry, today's a migraine day :)